turbo/engineapi/engine_server.go

```diff
@@ -166,6 +166,7 @@ func (s *EngineServer) newPayload(ctx context.Context, req *engine_types.Executi
 	if version >= clparams.DenebVersion {
 		header.BlobGasUsed = (*uint64)(req.BlobGasUsed)
 		header.ExcessBlobGas = (*uint64)(req.ExcessBlobGas)
+		header.ParentBeaconBlockRoot = parentBeaconBlockRoot
```

Comment:
Pls merge the latest `devel` into the PR so that this change already made by PR #7955 is not shown.